modules/ROOT/pages/sysconfig-network-configuration.adoc

```diff
@@ -890,3 +890,43 @@ storage:
           master=bond0
           slave-type=bond
 ----
+=== Disabling Automatic Configuration on Ethernet Devices (DHCP/SLAAT)
+
+==== via Butane
+
+To disable automatic configuration on ethernet devices (DHCP/SLAAT) with no other matching connections, add this to your Butane file.
+
+[source, yaml]
+----
+storage:
+  files:
+    - path: /etc/NetworkManager/conf.d/noauto.conf
+      mode: 0644 
+      contents: 
+        inline: |
+          [main]
+          # Do not do automatic (DHCP/SLAAC) configuration on ethernet devices
+          # with no other matching connections.
+          no-auto-default=*
+----
+
+Adding `no-auto-default=*` to a config file in /etc/NetworkManager/conf.d/ disables automatic configuration of ethernet devices.
```

Comment:
Let's add some context there about why it may be necessary to disable autoconfiguration.

```suggestion
By default, FCOS will attempt to autoconfigure (DHCP/SLAAC) every interface with a cable plugged in. In some network environments this may not be desirable. It's possible to change this behavior of NetworkManager with a configuration file dropin:
```